src/plugins/intel_gpu/src/kernel_selector/cl_kernels/normalize_gpu_across_spatial_blocked_ref.cl

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2018-2022 Intel Corporation
```

Comment:
I've already discussed with @wilson-seok  it here https://github.com/openvinotoolkit/openvino/pull/12459#discussion_r959228415 
What we can do is to get rid from initial plain kernels at all and use "_blocked" ones as they are for all layouts, but I'm afraid it will lead to performance regression for plain layouts, I can try to change to a common kernel with global indexation and see  performance impact